net-engine/job-skin-builder
# [TD-40] update-cbw-tools-from-node-14-to-latest

## Related issues



- [TD-40](https://netengine-trial.atlassian.net/browse/TD-40)

## Description



Upgraded node packages, node and upgraded react for job skin builder

## Pre-deployment steps



## Post-deployment steps



## Type of change



- [ ] Request (code for a specific purpose that will not add a feature. e.g. one off Rake task)
- [X] Maintenance task (Rake tasks or changes that do not change logic that will affect the application behaviour)

## How to test


Try creating a job skin using the new builder, you should be able to see the iframe code same as in the previous versions.

Here's some dummy resources to assist filling the builder form.
- https://dummyimage.com/1200x630/9ab2c9/fff.jpg&text=SocialImage
- https://dummyimage.com/1500x430/e6af8d/fff.jpg&text=Banner
- https://dummyimage.com/100x43/9ab2c9/fff.jpg&text=Logo
create more dummy images by altering the size and text

## Checklist

- Manual tests
  - [X] Changes were tested locally by the author
  - [ ] Changes were not tested and the reason was explained in the "How to test" section above

- Others
  - [ ] Client Success Team (for Scout) or client (for Agency) was notified before deployment if this PR involves risky changes; e.g.: running Rake tasks for modifying or deleting resources


[TD-40]: https://netengine-trial.atlassian.net/browse/TD-40?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
